ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java

```diff
@@ -459,6 +459,10 @@ private APIGatewayProxyResponseEvent redirectToFrontendErrorPage(String errorPag
 
     private boolean rpRequestedReturnCode(
             ClientRegistry clientRegistry, AuthenticationRequest authRequest) {
+        if (authRequest.getOIDCClaims() == null
```

Comment:
The changes to `IPVCallbackHandler` look like they should belong in a separate commit